en/flying/led_meanings.md

```diff
@@ -0,0 +1,38 @@
+# LED meanings
+
+[Pixhawk-series flight controllers](../flight_controller/pixhawk_series.md) have an RGB LED to indicate the current status of the vehicle. The diagram below shows the relationship between LED and vehicle status.
+
+![LED meanings](../../images/led_meanings.gif)
+
+> **Note:** **GPS Lock is required to takeoff** 
+The LED will only turn green (ready to execute guided mission) if PX4 is able to validate the global position. If the LED does not change from blue to green, verify that the GPS module is properly attached, Pixhawk is reading your GPS properly, and that the GPS is sending a proper GPS position.
```

Comment:
So this is much better. Is there any guidance on *how* you would verify that pixhawk is reading the GPS properly/GPS is sending proper GPS position? Would be good to link to that, since I have no idea how you'd test.